setup/common/003_solaris_cert_fix.rb

```diff
@@ -71,24 +71,25 @@
 EOM
 
 hosts.each do |host|
-  create_remote_file(host, "DigiCertTrustedRootG4.crt.pem", DigiCert)
-  on(host, 'chmod a+r /root/DigiCertTrustedRootG4.crt.pem')
-  on(host, 'cp -p /root/DigiCertTrustedRootG4.crt.pem /etc/certs/CA/')
-  on(host, 'rm /root/DigiCertTrustedRootG4.crt.pem')
+  next unless host.platform =~ /solaris-11(\.2)?-(i386|sparc)/ 
+    create_remote_file(host, "DigiCertTrustedRootG4.crt.pem", DigiCert)
```

Comment:
I don't think you need to indent following a next

Reply:
Thanks, updated